### ISSUE TITLE
How can I build for osx?

### ISSUE THREAD

**Issue by cevek** [powturbo/TurboRLE#4]
I have error on run `make`
```
trle.c:29:10: fatal error: 'malloc.h' file not found
```

**Comment by powturbo** [comment]
I've changed trle.c, please try again.

**Comment by cevek** [comment]
I think, you need to do it by yourself, because errors happen one after another :)
Next error:
```
error: use of undeclared identifier 'CLOCK_MONOTONIC'
```

I've add stackoverflow code:
```
#define CLOCK_REALTIME 0
#define CLOCK_MONOTONIC 0
// clock_gettime is not implemented on older versions of OS X (< 10.12).
// If implemented, CLOCK_REALTIME will have already been defined.
int clock_gettime(int clk_id, struct timespec* t) {
    struct timeval now;
    int rv = gettimeofday(&now, NULL);
    if (rv) return rv;
    t->tv_sec  = now.tv_sec;
    t->tv_nsec = now.tv_usec * 1000;
    return 0;
}
```

Next error: 
```
cc -O3 -march=native -minline-all-stringops trle.c -c -o trle.o
cc trle.o trlec.o trled.o -lrt -o trle
ld: library not found for -lrt
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [trle] Error 1
```

**Comment by powturbo** [comment]
Unfortunately, I've not osx machine for testing. TurboRLE was tested with gcc, clang on linux and mingw on windows.
